README.md

```diff
@@ -1,27 +1,123 @@
-# Deploy Google Cloud Virtual Machines in a public-only single region with Terraform
+# Deephaven create Image steps
 
-The script will install deephaven application on the virtual machines.
+#1. Login to GCP console and Click to cloud shell where we will run all the commands
 
-network-firewall.tf --> Configure basic firewall for the network
 
-network.tf --> Define network, vpc, subnet, icmp firewall
 
-provider.tf --> Configure Google Cloud provider
+   To set the project property in the core section, run:
+```
+        gcloud config set project YOUR_PROJECT_NAME
+```
 
-terraform.tfvars --> Defining variables
+    To set the zone property in the compute section, run:
+```
+        gcloud config set compute/zone us-central1-a
+```
 
-variables-auth.tf --> Application and authentication variables
+#Setup up project name and zone as environment variable as well
 
-vm-output.tf --> Output of VM
+```
+export PROJECT_ID="YOUR_PROJECT_NAME"
+export ZONE="us-central1-a"
+```
 
-vm.tf --> Create a Ubuntu VM
+#2. Run below command in the cloud shell which will create a GCP vm
 
-# Notes
+```
+gcloud beta compute --project=$PROJECT_ID instances create deephaven --zone=us-central1-a --machine-type=n2-standard-4 --subnet=default --network-tier=PREMIUM --maintenance-policy=MIGRATE --service-account=1088976619727-compute@developer.gserviceaccount.com --scopes=https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append --tags=deephaven,http-server,https-server --image=centos-7-v20210701 --image-project=centos-cloud --boot-disk-size=30GB --boot-disk-type=pd-balanced --boot-disk-device-name=deephaven --no-shielded-secure-boot --shielded-vtpm --shielded-integrity-monitoring --reservation-affinity=any
```

Comment:
Is "service-account" specific to your infrastructure?

Reply:
Yes. I have updated this command in test-himantej branch. 
You need to replace the PROJECT_ID and PROJECT_NUMBER in that command, instructions are updated as well accordingly.